README.md

```diff
@@ -1,16 +1,144 @@
-# Title
+# Default / Rest / Spread as Function Arguments
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Use JavaScript's default function argument as a parameter in a function
+- Use JavaScript's `rest` parameter as a parameter in a function
+- Use JavaScript's `spread` operator as a parameter in a function
 
 ## Introduction
 
-## SWBAT 1
+ES2016 has brought a number of improvements to parameters that we can pass into
+functions. Unlike most modern programming languages like Ruby or Python,
+JavaScript didn't have a way to pass a default parameter into a function.
+However, with the introduction of ES2016, that is no longer the case. It also
+introduced two other helpful parameters, the `rest` parameter and the `spread`
+operator. In this lab, we're going to learn about and practice using all three. 
```

Comment:
For the introduction, I think it might work better if we cut the first sentence and begin with the problem statement that JavaScript historically didn't have this capability, then move into the ES2016 solution.